src/main/java/org/folio/service/inventory/InventoryItemStatusSyncService.java

```diff
@@ -0,0 +1,147 @@
+package org.folio.service.inventory;
```

Comment:
Can you please format the file? IDEA updates javadocs for me.

Reply:
reformatted